plugins/repository-s3/src/main/java/org/opensearch/repositories/s3/S3BlobContainer.java

```diff
@@ -229,34 +245,29 @@ private void doDeleteBlobs(List<String> blobNames, boolean relative) throws IOEx
                 deleteRequests.add(bulkDelete(blobStore.bucket(), partition));
             }
             SocketAccess.doPrivilegedVoid(() -> {
-                AmazonClientException aex = null;
+                RuntimeException aex = null;
```

Comment:
```suggestion
                SdkException aex = null;
```

Reply:
Leaving this open based on https://github.com/opensearch-project/OpenSearch/pull/7372#discussion_r1205957185, we can change it if we take a decision to catch `SdkException` in all places.

Reply:
I/we need to test what happens on failure. If you have time, go for it? I should get to this Tuesday otherwise (Monday is a holiday).

Reply:
> I should get to this Tuesday otherwise (Monday is a holiday).
Sure, I should have time on Monday, will try to do some testing